3기/LeeGaYeon/week1/page1/style.css

```
@@ -0,0 +1,87 @@
+body {
+  /* font */
+  font-family: Spoqa Han Sans Neo;
+  font-size: 20px;
+  font-weight: 400;
+  line-height: 25px;
+  letter-spacing: 0em;
+  text-align: left;
+
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  min-height: 100vh;
+  height: 100vh;
```

```suggestion
  min-height: 100vh;
```

최소 높이만 넣어주시는게 좋아요! `height: 100vh;`를 넣게되면 높이를 100 viewport height로 "제한" 하겠다는 의미가 되어버려요.